shopelectro/tests/tests_js_analytics.py

```diff
@@ -100,6 +99,53 @@ def get_goals(self) -> selenium.Goals:
         goals.fetch()
         return goals
 
+    def assert_add(self, product: Product, goal_position: int):  # Ignore CPDBear
+        reached_goals = self.get_goals()
+        self.assertTrue(reached_goals)
+        reached = reached_goals[goal_position]
+
+        self.assertIn('add', reached)
+        self.assertEqual(reached['currencyCode'], 'RUB')
+
+        reached_detail = reached['add']  # Ignore CPDBear
+        self.assertEqual(
+            len(reached_detail['products']),
+            1,
```

Comment:
you assume that reached goal will be only one, right?
What if we'll use the method like this?
```
self.assert_add(product, 0)
self.assert_add(product, 1)
```

The same is for remove

Reply:
`reached_detail['products']` will have length greater 1 only in case if we will add/remove several products to the cart **at the same time**

Reply:
I'll fix it in the next issue

Reply:
> at the same time

@artemiy312 , y, my code example was exactly about adding at the same time. Our you mean something else?

Reply:
@duker33 Yes, I meant at the same moment.

Adding/Removing products with individual clicks publishes separate reached goals with length of `products` field =1

To get length of goal's `products` field >1 we need to add/remove several different products in one click. Right now the only way to do it is clearing a fulfilled cart.